src/SubsiteLoader.jsx

```diff
@@ -18,18 +18,21 @@ const SubsiteLoader = ({ content }) => {
   const subsiteEndpoint = content?.['@components']?.subsite
     ? content['@components'].subsite['@id']
     : null;
+  const subsiteState = useSelector((state) => state.subsite);
   const subsite = useSelector((state) => state.subsite?.data);
   const location = useLocation();
 
   useEffect(() => {
-    if (subsiteEndpoint && !subsite?.loading) {
-      dispatch(getSubsite(flattenToAppURL(subsiteEndpoint)));
+    if (subsiteEndpoint) {
+      if (!subsiteState?.loadingResults && subsiteState?.data !== null) {
```

Comment:
va bene ma ho un dubbio nel caso in cui si passi da un sottosito a un altro.. 
Penso che dovremmo fare le subrequest o controllare almeno che subsiteState.data['@id'] != subsiteEndpoint.

Altra cosa: non ho capito bene perchè fai la chiamata se subsiteState.data !=null , non dovresti farla se è null?

Reply:
ma... subsiteState.data !=null vuol dire che ti è già arrivato qualcosa..